src/app/core/entity-components/entity-list/entity-list.component.spec.ts

```diff
@@ -96,32 +117,27 @@ describe("EntityListComponent", () => {
           { provide: ExportService, useValue: {} },
           { provide: EntitySchemaService, useValue: mockEntitySchemaService },
           { provide: AttendanceService, useValue: mockAttendanceService },
+          { provide: ActivatedRoute, useValue: mockActivatedRoute },
         ],
       }).compileComponents();
     })
   );
 
-  beforeEach(() => {
-    fixture = TestBed.createComponent(EntityListComponent);
-    component = fixture.componentInstance;
-    component.listConfig = testConfig;
-    component.entityConstructor = Child;
-    component.ngOnChanges({
-      allEntities: new SimpleChange(null, component.allEntities, false),
-      listConfig: new SimpleChange(null, component.listConfig, false),
-    });
-    fixture.detectChanges();
-  });
-
   it("should create", () => {
+    createComponent();
```

Comment:
Maybe a better solution here is to have two scenarios (two `describe` blocks within the outer describe block). One would be a 'normal' setup and one a 'config setup'. You can then have two different `beforeEach` blocks in each `describe` block. Maybe have a look if that works and makes things a little clearer?